packages/api/src/posthog.ts

```diff
@@ -0,0 +1,11 @@
+import { PostHog } from "posthog-node";
+
+import { env } from "@blobscan/env";
+
+let posthog: PostHog | null = null;
+
+if (env.POSTHOG_ID && env.POSTHOG_HOST) {
```

Comment:
This is not consistent with the frontend. I prefer POSTHOG_HOST="https://us.i.posthog.com" as default here too.